### DISCUSSION TITLE
Allow To Do lists to be reordered

### DISCUSSION THREAD

**Discussion by SamBouwer** [home-assistant/frontend#18507] [Other feature requests]
We can reorder items in the sidebar via the user settings > "Change the order and hide items from the sidebar" and we can reorder items within a to do list. It would be nice to be able to reorder the to do lists themselves in the "To-do lists" list.

**Comment by jonpittam** [comment]
I really hope this gets added, I have a few to do lists but I want to set a shopping one I've created to be the first one that opens.

**Comment by frenck** [comment]
This has been available/added

![CleanShot 2024-01-15 at 11 25 25@2x](https://github.com/home-assistant/frontend/assets/195327/7c48f01d-565d-4502-9c2b-97e57dc1417b)

**Reply by SamBouwer** [reply]
Is it also possible to reorder the tasks lists themselves? (so not tasks within a list, so for example in your print screen put Picnic at the top?)

**Reply by ronlut** [reply]
@frenck I stumbled upon this issue, I see it's closed but I don't see a way to do it yet.. The purpose it to redorder the lists themself, not the items in each list.
Is this possible?
Thanks!
